gossip/hub/hub_server/main.go

```diff
@@ -89,10 +89,18 @@ func main() {
 
 	dialOpts := []grpc.DialOption{grpc.WithInsecure()}
 	if strings.Contains(*rpcBackend, ",") {
-		glog.Infof("Using FixedBackendResolver")
-		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
-		res := ctfeutil.FixedBackendResolver{}
-		dialOpts = append(dialOpts, grpc.WithBalancer(grpc.RoundRobin(res))) // nolint: megacheck
+		// This should probably not be used in production. Either use etcd or a gRPC
```

Comment:
Could potentially just delete this support for passing in multiple backends?
It's only meant to be an example.

Reply:
True, it probably wasn't being used. I've removed it.